config/routes.rb

```diff
@@ -1,8 +1,28 @@
 Rails.application.routes.draw do
+  devise_for :users
+
+  devise_scope :user do
+    get '/users/sign_out' => 'devise/sessions#destroy'
```

Comment:
oof, you sure you need this? I don't want to dig into devise gocs right now, but the gem should cover this case for sure